samples/nrf9160/aws_fota/src/main.c

```diff
@@ -65,6 +65,55 @@ void bsd_irrecoverable_error_handler(uint32_t err)
 
 #endif /* defined(CONFIG_BSD_LIBRARY) */
 
+/* Topic for updating shadow topic with version number */
+#define AWS "$aws/things/"
+#define UPDATE_DELTA_TOPIC AWS "%s/shadow/update"
+#define SHADOW_STATE_UPDATE \
+"{\"state\":{\"reported\":{\"nrfcloud__dfu_v1__app_v\":\"%s\"}}}"
+
+static int update_device_shadow_version(struct mqtt_client *const client)
+{
+	struct mqtt_publish_param param;
+	char update_delta_topic[strlen(AWS) + strlen("/shadow/update") +
```

Comment:
Can use `sizeof - 1` instead to avoid calculations runtime

Reply:
Not in this PR :)